lua/rustaceanvim/lsp/init.lua

```diff
@@ -304,12 +355,14 @@ local function rust_analyzer_cmd(opts)
     M.restart()
   elseif cmd == RustAnalyzerCmd.reload_settings then
     M.reload_settings()
+  elseif cmd == RustAnalyzerCmd.target then
+    M.set_target_arch(nil, arch)
   end
 end
 
 vim.api.nvim_create_user_command('RustAnalyzer', rust_analyzer_cmd, {
   nargs = '+',
-  desc = 'Starts or stops the rust-analyzer LSP client',
+  desc = 'Starts, stops the rust-analyzer LSP client or change target',
```

Comment:
suggestion: It would be nice to have completions for valid targets. But maybe for anther time - I'm happy to merge and create an issue for completions later.

Reply:
Yeah that should be later..I needed to get this up quick as it was affecting my workflow :xD